src/Hyperwallet/Hyperwallet.php

```diff
@@ -420,6 +422,21 @@ public function createTransfer(Transfer $transfer) {
         return new Transfer($body);
     }
 
+    public function createTransferRefund($transferToken, TransferRefund $transferRefund = null) {
```

Comment:
TransferRefund $transferRefund = null can be just TransferRefund $transferRefund

Reply:
This is to accept null $transferRefund value. However when null value is passed, proper user friendly error message is returned. Removing 'null' value fails the test.

Reply:
I still feel that = null should be removed.

Reply:
Done.